pvlib/temperature.py

```diff
@@ -900,7 +904,7 @@ def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
 
     wind_speed = wind_speed.values
     P = a[0] + a[1]*wind_speed + a[2]*unit_mass + a[3]*wind_speed*unit_mass
-    timedeltas = np.arange(window, 0, -1) * time_step
+    timedeltas = np.arange(window, 0, -1) * (time_step*60)  # s to min
```

Comment:
If `timestep` is in seconds, then you would have to divide by 60 to get minutes...

Reply:
```suggestion
    timedeltas = np.arange(window, 0, -1) * (time_step*60)  # min to s
```